canonical/microstack.run
# Refresh homepage

## Done

**BLOCKED** by [this brand issue](https://warthogs.atlassian.net/browse/WD-1195)
- Refreshes homepage based on changes in the [copydoc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- An issue has been created to address Lyubo's comments on inconsistant stroke thickness that can be tracked [here](https://warthogs.atlassian.net/browse/WD-1195)

## QA

- Go to https://microstack-run-214.demos.haus/ and compare against [copydoc](https://docs.google.com/document/d/1-QO5zAclj469qGvFmNJBjQ_qqI_T_uliwj6mAo8ZbCs/edit#)
- Check that all six individual tabs work and don't interfere with each other


## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6311
